python-checks/src/test/resources/checks/hotspots/cors/cors.py

```diff
@@ -138,8 +138,7 @@ def flask_add_headers_no_args():
 @app.route("/")
 def flask_make_response_headers_set():
     resp = make_response("hello")
-    resp.headers['Access-Control-Allow-Origin'] = '*' # Noncompliant
-#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    resp.headers['Access-Control-Allow-Origin'] = '*' # FN
```

Comment:
Do you know why this becomes a FN? Is this the one related to the other ticket you created?

Reply:
This is because attributes were missing from typeshed classes. I think this change is not needed after I rebase on master :)

Reply:
I confirm, after rebase the issue is raised again :)